luarules/gadgets/unit_collision_damage_behavior.lua

```diff
@@ -24,12 +24,15 @@ local collisionVelocityThreshold = 99 / Game.gameSpeed
 local validCollisionAngleMultiplier = math.cos(math.rad(20)) --degrees
 
 -- Decrease this value to make units move less from impulse. This defines the maximum impulse allowed, which is (maxImpulseMultiplier * mass) of each unit.
-local maxImpulseMultiplier = 165 / Game.gameSpeed
+local maxImpulseMultiplier = 5.5
```

Comment:
?

Reply:
That's a ratio, not a velocity derivative. It reduces impulse. Has nothing to do with game speed so I took out that part.

Reply:
right